### DISCUSSION TITLE
V2box Connection Issue

### DISCUSSION THREAD

**Discussion by shcalay** [hiddify/Hiddify-Manager#3694] [Q&A سوالات رایج]
I had V2box set up on my iPhone, and it was working fine until yesterday. However, I am now unable to slide to connect on the home page. When dragging it to the right, it immediately goes back to the left without establishing a connection. I have good pings on all my configurations. Despite deleting the app, reinstalling it, and setting up the configurations from scratch, the problem persists. Can you assist me with resolving this issue, please? Many Thanks

**Comment by MoriKermani** [comment]
Hi, have you chosen the current config through the config list? The left side color of the config should be different.
